frontend/src/lib-customizations/espoo/citizen/assets/i18n/sv.tsx

```diff
@@ -70,8 +71,20 @@ const sv: Translations = {
     title: 'Enheter på kartan',
     mainInfo:
       'I den här vyn kan du på kartan söka enheter med småbarnspedagogik och förskola.',
+    privateUnitInfo: function PrivateUnitInfo() {
+      return (
+        <span>
+          För information om privata daghem,{' '}
+          <ExternalLink
+            text="klicka här."
+            href="https://www.esbo.fi/sv-FI/Utbildning_och_fostran/Smabarnspedagogik/Privat_smabarnspedagogik"
```

Comment:
This leads to english page about the swedish services

Reply:
You may have auto-translate swedish on?